django/db/models/manager.py

```diff
@@ -87,8 +86,7 @@ def manager_method(self, *args, **kwargs):
 
         new_methods = {}
         # Refs http://bugs.python.org/issue1785.
```

Comment:
Did you check if this comment is still relevant? If so, a better explanation would be useful, though not needed as part of this PR.